packages/console/package.json

```diff
@@ -90,20 +90,28 @@
     "rules": {
       "complexity": "off",
       "@typescript-eslint/prefer-nullish-coalescing": "off"
-    }
+    },
+    "overrides": [
+      {
+        "files": [
+          "*.test.ts"
+        ],
+        "rules": {
+          "@typescript-eslint/no-unsafe-return": "off"
+        }
+      },
+      {
+        "files": [
+          "*.d.ts"
+        ],
+        "rules": {
+          "@typescript-eslint/consistent-type-definitions": "off"
```

Comment:
Should this be turned off in ts-config? Seems like we don't want to limit the usages of `type` and `interface`, though we do prefer `type` over `interface` in practice

Reply:
personally i'd like to have this

Reply:
But I just checked our eslint-config and seems it's already in there?

<img width="728" alt="image" src="https://user-images.githubusercontent.com/12833674/182844882-9263d15c-b2af-490e-8f20-8f5553e6ba0e.png">

Reply:
yes, it's another mistake. should be for testing only. will remove